### ISSUE TITLE
Camera position and rotation?

### ISSUE THREAD

**Issue by N3RDIUM** [Magoninho/3D-projection-tutorial#1]
##### I really need help with just that in my repo too: [LINK](https://github.com/1upCommunity/Py3D)

**Comment by Magoninho** [comment]
Oh, hello there!
Well, this project is becoming older and I really don't even remember how I did it. I was trying to implement a lot of stuff at the time, camera was one of these things. But I decided that was enough and abandoned this project, and then, forgot about everything.

And now, as I can see, you have a big project that implements my code. You really have a stronger profile, with a lot of cool stuff on it. And I am just a little guy that doesn't have a lot of algebra skills and was trying to make a simple rotating cube with python, nothing big at all.

So, I think it will take a long long time (or even never idk) to implement camera position and rotation to the engine. I really wish I could help you, but I'm on the last year of high school (in Brazil this means suffering), and really out of time for technology now :(

Sorry
